tucan/io.py

```diff
@@ -24,10 +24,21 @@ def graph_from_file(filepath: str) -> nx.Graph:
     filepath = Path(filepath)
     if filepath.suffix != ".mol":
         raise IOError(f"The file must be in '.mol' format, not {filepath.suffix}.")
-    filecontent = _read_file(filepath)
+    with open(filepath) as file:
+        filecontent = file.read()
+
+    return graph_from_molblock(filecontent)
+
+
+def _split_into_tokenized_lines(string: str) -> List[List[str]]:
+    lines = [line.rstrip().split(" ") for line in string.splitlines()]
+    return [[value for value in line if value != ""] for line in lines]
+
 
-    atom_props = _parse_atom_block_molfile3000(filecontent)
-    bonds = _parse_bond_block_molfile3000(filecontent)
+def graph_from_molblock(molblock: str) -> nx.Graph:
```

Comment:
The naming could be misleading, it's not the molblock (i.e., Ctab, see, page 20 in https://discover.3ds.com/sites/default/files/2020-08/biovia_ctfileformats_2020.pdf) that's passed in, but the entire file, right?

Reply:
I used the naming from rdkit, which has the [Chem.MolFromMolBlock](https://www.rdkit.org/docs/source/rdkit.Chem.rdmolfiles.html#rdkit.Chem.rdmolfiles.MolFromMolBlock) and [Chem.MolToMolBlock](https://www.rdkit.org/docs/source/rdkit.Chem.rdmolfiles.html#rdkit.Chem.rdmolfiles.MolToMolBlock) functions. `Chem.MolToMolBlock(mol)` returns a valid molfile (header+CTAB) as string.
I'm not sure about renaming at the moment. _graph_from_string_ would be more confusing. Maybe _graph_from_molfile_text_? (naming from InChI's MakeINCHIFromMolfileText())

Reply:
The current choice `graph_from_molfile_text` feels less wrong than `graph_from_molblock`. Although there's of course a point to be made for sticking with known conventions, i.e., calling it `graph_from_molblock` since that's what's used over at RDKit. As far as I'm concerned we can stick with `graph_from_molfile_text` for now.